client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -249,9 +220,22 @@ class ActivityLogItem extends Component {
 	}
 
 	renderHeader() {
+		const { actor = {} } = this.props.log;
+		const {
```

Comment:
I wasn't sure the best way of going about this.

At a high level, I've been trying to keep the sub-components pure and simple, relying only on flat simple props (not objects) whenever possible. In this view `ActivityLogItem` knows how to pass the relevant pieces of its props to it's sub-components which will handle more of the rendering logic.

Internally, I expect `ActivityActor` to gracefully deal with this props or warn appropriately.

Does this all seem reasonable and worthwhile?

For this case specifically, it might be better as a series of `_.get`s.